specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,710 @@
+              "schema": {
+                "type": "string"
+              }
+            }
+          ],
+          "requestBody": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/PayloadMessage"
+                }
+              }
+            }
+          },
+          "responses": {
+            "202": {
+              "description": "Accepted"
+            }
+          }
+        }
+      },
+      "/api/v1/hubs/{hub}/users/{id}": {
+        "post": {
+          "description": "Broadcast a message to all clients within the target group.",
+           "tags": [],
+           "parameters": [
+            {
+              "name": "hub",
+              "in": "path",
+              "required": true,
+              "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+              "schema": {
+                "type": "string"
+              }
+            },
+            {
+              "name": "id",
+              "in": "path",
+              "required": true,
+              "description": "Target user Id.",
+              "schema": {
+                "type": "string"
+              }
+            }
+          ],
+          "requestBody": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/PayloadMessage"
+                }
+              }
+            }
+          },
+          "responses": {
+            "202": {
+              "description": "Accepted"
+            }
+          }
+        }
+      },
+      "/api/v1/hubs/{hub}/connections/{connectionId}": {
+        "post": {
+          "description": "Send message to a single connection Id.",
+          "tags": [],
+          "parameters": [
+            {
+              "name": "hub",
+              "in": "path",
+              "required": true,
+              "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+              "schema": {
+                "type": "string"
+              }
+            },
+            {
+              "name": "connectionId",
+              "in": "path",
+              "required": true,
+              "description": "Target connection Id.",
+              "schema": {
+                "type": "string"
+              }
+            }
+          ],
+          "requestBody": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/PayloadMessage"
+                }
+              }
+            }
+          },
+          "responses": {
+            "202": {
+              "description": "Accepted"
```

Comment:
Does your 202 have a way to poll the result that is compatible with Microsoft OenAPI guidelines?

Reply:
No, for sending messages `202` does not have corresponding polls, it **fire and forget**. Sending messages through REST API, including *send to connection, send to user/group and broadcast* follows **fire and forget** pattern. The service does not track if every subscriber has recieved the message and there is no plan to support that yet. (Customers might implement their own message protocol if they want to guarantee message delivery, for example using acks)